servicenow/plugin.spec.yaml

```
@@ -401,3 +401,27 @@ triggers:
         description: JSON object representing changed fields (map of field name to previous and current values)
         type: object
         required: true
+  incident_created:
+    title: Incident Created
+    description: Identifies if a new incident has been created
+    input:
+      query:
+        title: Query
+        description: Non-encoded query string to match incident records
```

```suggestion
        description: Non-encoded query string to match incident records
```
Can you add what will happen if the user omits the query string? What is the behavior?